README.md

```diff
@@ -376,6 +432,22 @@ encoded BibTeX file (with the citation keys serving as local Source IDs). The fi
 - or specified as path relative to the metadata file given for top-level common property `dc:source` in the dataset's metadata.
 
 
+### Compressed data or reference files
+
+CLDF datasets may contain large data or reference files which may be inconvenient (e.g. because the size exceeds GitHub's
+100MB filesize limit). In such cases, the dataset creator may compress individual files using the
+[ZIP format](https://en.wikipedia.org/wiki/ZIP_(file_format)) (which works really well on CSV and BibTeX files).
+The resulting ZIP archive MUST contain the zipped file as single member and MUST be named after the original
+file, adding `.zip` as filename extension. The filename references in the metadata MUST be kept unchanged.
+
+CLDF processing software MAY implement zip-file discovery, i.e. if a filename referenced in the metadata cannot
+be found, but a file `filename.zip` is found, processing MUST proceed with the unzipped content of `filename.zip`.
+
```

Comment:
should this be filename.ext.zip? e.g. we have `posterior.trees.zip` in phlorest and `sources.bib.zip` in e.g. glottolog

Reply:
I'll rewrite to `filename_with_extension.zip` for maximum transparency. For me, "filename" already includes the extension - while "stem" is the filename with the - first level of - extension(s) split off. But I guess that intuition may be different in the windows world.